src/dxgi.rs

```diff
@@ -73,6 +74,7 @@ crate::weak_com_inheritance_chain! {
         Factory4(dxgi1_4::IDXGIFactory4), from_factory4, as_factory4, unwrap_factory4;
         Factory5(dxgi1_5::IDXGIFactory5), from_factory5, as_factory5, unwrap_factory5;
         Factory6(dxgi1_6::IDXGIFactory6), from_factory6, as_factory6, unwrap_factory6;
+        FactoryMedia(dxgi1_3::IDXGIFactoryMedia), from_factory_media, as_factory_media, unwrap_factory_media;
```

Comment:
This doesn't do what you think it does. This means that if `dxgi1_3::IDXGIFactoryMedia` than all previous versions must also exist. There's no guarantee that dxgi1_6 is supported when IDXGIFactoryMedia is

Reply:
Ok, let me fix it.

Reply:
It looks like [IDXGIFactoryMedia](https://learn.microsoft.com/en-us/windows/win32/api/dxgi1_3/nn-dxgi1_3-idxgifactorymedia) inherits directly from [IUnknown](https://learn.microsoft.com/en-us/windows/desktop/api/unknwn/nn-unknwn-iunknown).